src/components/PopupWithForm.js

```diff
@@ -15,19 +30,42 @@ export default class PopupWithForm extends Popup {
     });
     return inputValues;
   }
-  // close() {
-  //   this._popupForm.reset();
-  //   super.close();
-  // }
+
+  open() {
+    this._popupForm.reset();
+    super.open();
+  }
 
   setEventListeners() {
     //event listeners
-    this._popupForm.addEventListener("submit", (e) => {
+    this._popup = this._popupForm = this._popupElement.querySelector(".modal__form");
```

Comment:
`    this._popup = this._popupForm = this._popupElement.querySelector(".modal__form");
`

` this._popup =` is not needed

You need to define only `this._popupForm`


```js

    this._popupForm = this._popupElement.querySelector(".modal__form");
    this._popupForm.addEventListener("submit", (e) => {
```